lib/sorcery/model/submodules/reset_password.rb

```diff
@@ -124,10 +124,14 @@ def reset_password_reset_page_access_counter
           end
 
           # Clears token and tries to update the new password for the user.
-          def change_password!(new_password)
+          def change_password(new_password, raise_on_failure: false)
```

Comment:
For default values, I've normally seen `def method(variable = value)` rather than `def method(variable: value)`, are you sure this works as intended?